README.md

```diff
@@ -3,14 +3,17 @@
 # Bitcoin Hashes Library
 
 This is a simple, no-dependency library which implements the hash functions
-needed by Bitcoin. These are SHA1, SHA256, SHA256d, SHA512, and RIPEMD160. As an
-ancilliary thing, it exposes hexadecimal serialization and deserialization,
-since these are needed to display hashes anway.
+needed by Bitcoin. These are `Sha256`, `Sha256d` (double SHA-256), `Sha256t`
+(tagged SHA-256), `Sha512`, `Ripemd160` and `Hash160` (representing 
```

Comment:
Why suddenly remove the caps? The algorithms are usually referred to in all-caps..